aries-site/src/pages/components/checkbox.mdx

```diff
@@ -36,8 +37,21 @@ Checked and unchecked states refer to when an individual CheckBox is checked or
 
 ### CheckBox outside of FormField
 
```

Comment:
Thinking that we should include a brief description of what the visual changes are for a checkbox outside of a formfield here...

"CheckBox can also be used outside of a FormField. When doing so, the CheckBox does not have a border or label. Here are some common use cases for when a CheckBox can be used outside of a FormField: "

Reply:
Agreed ^^

Reply:
done